distribution/lib/Standard/AWS/0.0.0-dev/src/Main.enso

```diff
@@ -0,0 +1,7 @@
+import project.Database.Redshift.Redshift_Details.Redshift_Details
+import project.Database.Redshift.Redshift_Details.AWS_Credential
+
+export project.Database.Redshift.Redshift_Details.Redshift_Details
+export project.Database.Redshift.Redshift_Details.AWS_Credential
+
+from project.Database.Redshift.Redshift_Details.Redshift_Details export Redshift
```

Comment:
Let's remove this - I think we might remove the others in due course.

Reply:
We agreed on Discord to tackle this later.